postcss.config.js

```diff
@@ -1,5 +1,15 @@
 module.exports = {
   plugins: {
     'postcss-preset-mantine': {},
+    'postcss-prefix-selector': {
```

Comment:
Were we able to figure out the issues with `data-mantine-*` attribute?

Reply:
Currently this plugin can't fix the `data-mantine-*` attribute issue, as I searched the mantine source code `data-mantine-color-scheme` are hardcoded in some parts

Reply:
For this issue I have opened a discussion in https://github.com/orgs/mantinedev/discussions/6573

Reply:
@arjunvegda What do u think if we merge this fix first and provide a fix for `data-mantine-*` attribute issue later?

Reply:
Does the dark/light mode + everything else work as expected with this change?

Reply:
hey aj current dark/light mode not working due the the `data-mantine-*` issue, others work as expected.

Reply:
In that case, would it be safer to wait? 😅 Theming is one of the important features of Jotai DevTools.

Reply:
Hey aj, sure. Meanwhile I will be looking at other solutions.

Reply:
Wondering if it would be worth it to simply migrate away from Mantine then?

Reply:
Yes I think we should migrate to another component library at this point

Reply:
Do we even need a component library here for the dev tools UI - since this isn't a web page that's meant to be responsive on all device screen sizes? This feels like something simple that we can achieve with just native HTML elements and CSS in React

Reply:
I guess we could use something like https://ui.shadcn.com/, which is headless component library but styled with tailwind.

Reply:
I agree with the sentiment, we should look into migrating away from Mantine. ShadCN is a good alternative, would anyone like to give it a shot? Ideally, we'd want to have 1:1 migration (with theme, look, etc.) with zero impact on the userland.

Reply:
Not sure if we even need tailwind (not that I'm against using it, or other CSS libraries) -- as long as the devtools can be styled with dark and light theme support, it should be sufficient.

Reply:
> Not sure if we even need tailwind (not that I'm against using it, or other CSS libraries) -- as long as the devtools can be styled with dark and light theme support, it should be sufficient.

Certainly! Here's a rewritten version:

But using shadcn can reduce the overhead of migration. A great thing is that it is built on top of Radix, meaning it’s unstyled, so breaking changes are less likely to occur.

Reply:
But using shadcn can reduce the overhead of migration. A great thing is that it is built on top of Radix, meaning it’s unstyled, so breaking changes are less likely to occur.

Reply:
> I agree with the sentiment, we should look into migrating away from Mantine. ShadCN is a good alternative, would anyone like to give it a shot? Ideally, we'd want to have 1:1 migration (with theme, look, etc.) with zero impact on the userland.

Actually, I am interested in the migration task as I would like to make the same kind of devtool for zustand.

Reply:
Sorry, I've been a little occupied lately. Let me DM you later this week @Elvincth